etc/settings.cfg.sh

```diff
@@ -25,9 +25,7 @@ export EUPSPKG_SOURCE=${EUPSPKG_SOURCE:-git}
 export EUPS_PKGROOT=$LSSTSW/distserver/production
 
 # the location of source repositories
-BASE='git://git.lsstcorp.org/LSST'
-SBASE='https://stash.lsstcorp.org/scm'
-export REPOSITORY_PATTERN="$BASE/sims/%(product)s.git|$BASE/DMS/%(product)s.git|$BASE/DMS/devenv/%(product)s.git|$BASE/DMS/testdata/%(product)s.git|$BASE/external/%(product)s.git|$SBASE/sim/%(product)s.git"
+export REPOFILE=${LSSTSW}/etc/repos.yaml
```

Comment:
Nice catch.  I will repush in a moment.

What's the value of having a REPOSITORY_PATTERN if the location of every repository is explicitly mapped?  That could both hide errors in `repos.yaml` and be non-deterministic.  There's also some logic that may prevent a repos.yaml supplied url from changing the remote in an existing clone if its remote matches a patterns generated url.

I left the pattern functionality in because I'm unsure of how this package is being used, if at all, outside of lsst-dev.  I'd be happy to remove the functionality completely or add the ability to fetch a repos.yaml file via HTTP so any existing users wouldn't be required to supply a configuration file.  However, for our live build system we need to achieve two things:

1) Fully decouple the repository name from the EUPS package name.  We've already demonstrated this by renaming the github `pex_harness` repo to `legacy-pex_harness`.

2) Have deterministic git remotes; as it is now, a repository being created at the correct path could automatically supersede the desired repo (or not, untli the existing clone is deleted).